bin/civi-test-run

```diff
@@ -256,6 +271,7 @@ for TESTTYPE in $SUITES ; do
     phpunit-drupal) task_phpunit_drupal ;;
     phpunit-wordpress) task_phpunit_wordpress ;;
     phpunit-backdrop) task_phpunit_backdrop ;;
+    phpunit-creditnotes) task_phpunit_core_extension sequentialcreditnotes Financial ;;
```

Comment:
I'm not sure I like the file structure of "civicrm-core:CoreExtensions/Financial/sequentialcreditnotes". But I'm not criticizing this PR for that. However, if that changed, then you'd have to edit both the callers and content of the function.

One of these would feel like a cleaner separation of concerns to me:

```
## Give both full-key and full-path
task_phpunit_extension "sequentialcreditnotes" "$CIVI_CORE/CoreExtensions/Financial/sequentialcreditnotes"

## Just give a path. 
task_phpunit_extension "$CIVI_CORE/CoreExtensions/Financial/sequentialcreditnotes"
# note: The function can figure out the key using `cd $path; civix info:get -x @key`)

## Just give a key. 
task_phpunit_extension "sequentialcreditnotes"
# note: The function can figure out the path using `cv path -x $key`)
```

Reply:
I was rather hoping any phpunit.xml files under the CoreExtensions would be automagically picked up rather than us keeping adding them as we add new ones

Reply:
Sadly @eileenmcnaughton that is not the case even when we added in api_v4 we needed to add in another test suite function for it